state/state.go

```diff
@@ -909,12 +909,7 @@ func (st *State) FindEntity(tag names.Tag) (Entity, error) {
 		}
 		return model.Operation(tag.Id())
 	case names.CharmTag:
```

Comment:
Idk really what this function is used for and where, but my instinct is it would be better to keep it, so that in the future if we do want to use it with a charm tag it still works